test/integration/comments.test.js

```diff
@@ -29,13 +30,12 @@ const studioId = process.env.COMMENT_STUDIO_ID || 10005646;
 const studioUrl = `${rootUrl}/studios/${studioId}/comments`;
 
 // setup comments to leave
+// make sure they are unique and will not be censored (avoid numbers that might look like phone numbers or other PII)
 const date = new Date();
-const dateString = `Y:${date.getFullYear()} - M:${date.getMonth() + 1} - D:${date.getDate()} ` +
-`: ${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
-const buildNumber = process.env.CIRCLE_BUILD_NUM || dateString;
```

Comment:
On CCI, these build numbers were small, like `10`. On GHA, they're much larger numbers that our phone number filter blocked. It just needs to be unique, so an ISO date string with sub-second precision works well.